src/Components/WebAssembly/WebAssembly.Authentication/src/Services/AccessTokenProviderAccessor.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication.Internal
+{
+    internal class AccessTokenProviderAccessor : IAccessTokenProviderAccessor
```

Comment:
As with the comment about `IAccessTokenProviderAccessor`, I haven't spotted what this is used for. If you could clarify what consumes this feature that would be great.

Reply:
I answered [here](https://github.com/dotnet/aspnetcore/pull/20192/files#r402616175). The main goal is to make it easy to consume the `IAccessTokenProvider` from the UserFactory since that's a common thing and it's the lesser of evils.

Reply:
Understood now - thanks